src/js/stores/MetronomeStore.js

```diff
@@ -166,10 +166,40 @@ class MetronomeStore extends EventEmitter {
     MetronomeActions.runJob(jobID);
   }
 
-  suspendSchedule(jobID) {
-    let schedule = this.getJob(jobID).schedules[0];
-    schedule.enabled = false;
-    MetronomeActions.suspendSchedule(jobID, schedule);
+  disableSchedule(jobID) {
+    let job = this.getJob(jobID);
+    if (job == null) {
```

Comment:
The eslint `newline-before-return`-rule treats both as correct, so I am not sure how this should forced with eslint.
http://eslint.org/docs/rules/newline-before-return

Additionally I refer to Kenny's sentence.